templates/rhpam75-authoring-ha.yaml

```diff
@@ -219,12 +219,17 @@ parameters:
 - displayName: DataGrid Image
   description: DataGrid image.
   name: DATAGRID_IMAGE
-  value: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.1
+  value: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.2
+  required: true
+- displayName: DataGrid Container CPU Limit
+  description: DataGrid Container cpu limit.
+  name: DATAGRID_CPU_LIMIT
+  value: "4"
   required: true
 - displayName: DataGrid Container Memory Limit
   description: DataGrid Container memory limit
   name: DATAGRID_MEMORY_LIMIT
-  value: "512Mi"
+  value: "4Gi"
```

Comment:
I don't know all the background around this decision, but isn't 4gi too much? I mean, Business Central that is a way heavy than JDG uses 2Gi.

Reply:
@spolti Gotcha. @jakubschwan , @sutaakar : Do you think we can trim this down more? (Looking for another suggestion here, too, sorry.)